integration/setup/key.go

```diff
@@ -0,0 +1,31 @@
+// +build k8srequired
+
+package setup
+
+import (
+	"fmt"
+)
+
+func awsConfigReleaseName(clusterID string) string {
+	return fmt.Sprintf("e2esetup-awsconfig-%s", clusterID)
```

Comment:
Why do we need two packages with the exact same method and values ?
I am referring to `integration/key/release_name.go`

Reply:
Because I left it half way during the transition :) Thanks!